src/JointAdapter.cpp

```diff
@@ -87,6 +88,13 @@ void JointVelocityAdapter::update(
   if (std::isnan(pidVelocity))
     throw std::range_error("calculated pidVelocity is NaN");
 
+  if (desiredVelocity + pidVelocity > 5.0)
```

Comment:
Where is 5 coming from?

Reply:
Arbitrary, set by @gilwoolee

Reply:
@brianhou Since it is arbitrary, I made it a constructor argument with default (see next commit). Resolve if you think that's acceptable for now.

Reply:
Is there a way to take this from the URDF info instead?

Reply:
Yeah this was arbitrary; it was more of a temporary hack for `ada`. I think we should get it from URDF.